scripts/docker-compose-files.txt

```diff
@@ -4,6 +4,7 @@ docker-compose-odoo.yml
 docker-compose-odoo-sso.yml
 docker-compose-openmrs.yml
 docker-compose-openmrs-sso.yml
+docker-compose-bahmniemr.yml
```

Comment:
Remove this.
To run Ozone with Bahmnni instead of 03, one as to explicitly add it to `docker-compose-files.txt` and remove OpenMRS (O3) files.